iterative/utils/analytics_test.go

```diff
@@ -36,18 +38,20 @@ func TestUserId(t *testing.T) {
 		"user_id": userId,
 	}
 	json, _ := json.MarshalIndent(data, "", " ")
+	err := os.MkdirAll(filepath.Dir(old), 0755)
+	assert.Nil(t, err)
+	err = ioutil.WriteFile(old, json, 0644)
+	assert.Nil(t, err)
```

Comment:
```suggestion
	assert.NoError(t, err)
	err = ioutil.WriteFile(old, json, 0644)
	assert.NoError(t, err)
```

Reply:
Good catch, thank you!

Reply:
Also, if you like it, the check could be performed in the same line, e.g.

```go
assert.NoError(t, ioutil.WriteFile(old, json, 0644))
```